packages/docs/docs/charting/capturing-vital-signs.md

```diff
@@ -0,0 +1,136 @@
+  }
+}
+
+```
+
+In this example, we use the `code` field to define the vital we are measuring, in this case body temperature. We define it using a LOINC code, as well as a `display`, which is a readable representation of what we are measuring. In addition, since we are measuring temperature, which is easily represented as a number or quantity, we use the `valueQuantity` field to define the measurement result. We can use the `unit` field to provide a readable string of the unit of the measurement.
+
+## Who Performed the `Observation`
+
+Observation data can come from various sources, including the practitioner, a nurse, being self-reported by the patient, or many others. The source of the observation should be recorded on the `Observation.performer` element. This can be important to help provide a degree of confidence in the observation and to inform where any follow-up questions should be directed.
+
+## Medical Devices
+
+Medical devices, which can be represented by either the [`Device`](/docs/api/fhir/resources/device) or [`DeviceDefinition`](/docs/api/fhir/resources/devicedefinition) resources are able to provide data regarding patients, tests, etc. A `Device` is an individual instance of a device, whereas `DeviceDefinition` is a kind of device, which can have many instances. In this context, a device is defined as a manufactured item that is used in the provision of healthcare without being substantially changed through that activity.
+
+Devices can cover a wide range of equipment, such as durable or reusable medical equipment, as well as disposable equipment used for diagnostic, treatment, or research. Essentially, a device can range from a tongue depressor all the way to an MRI machine, and the fields in the `Device` resource are flexible enough to cover this range. Additionally, the `Device` resource does not necessarily refer to a strictly medical device. This can include wearable devices, such as smart phones or watches.
+
+It is important to note that an `Observation` can come from a device, and are represented by the `Observation.device` element. This element should reference a `Device` or a `DeviceMetric` resource from which the `Observation` resulted. For example, if a heart rate measurement is taken from a patient's watch, the resulting `Observation` should reference that watch.
+
+**Example: **
+
+```ts
+
+// A patient's watch that provides a measurement of their heart rate
+{
+  resourceType: 'Device',
+  id: 'example-watch',
+  /// ...
+}
+
+// The observation resulting from the watch
+{
+  resourceType: 'Observation',
+  id: 'example-observation-2',
+  // ...
+  code: {
+    system: 'http://loinc.org',
+    code: '8867-4',
+    display: 'Heart rate'
+  },
+  valueQuantity: {
+    value: 67,
+    unit: 'beats per minute'
+  }
+  // The device is a reference to the above watch
+  device: {
+    resource: {
+      resourceType: 'Device',
+      id: 'example-watch'
+    }
+  }
+}
+```
+
+## Reference Ranges
```

Comment:
We have a [bigger doc](https://medplum-www-git-rahul-documentation-lab-service-menu-medplum.vercel.app/docs/careplans/reference-ranges) on the way that goes into detail about to create reference ranges, so this section can be pretty light and just reference that doc. 

The important point here is that the `Observation.referenceRange` field should reference the _specific_ range from the `ObservationDefinition` that corresponds to what is considered normal for _this_ patient, given their age, gender, race, etc.